RUFAS/graph_generator.py

```diff
@@ -158,7 +156,8 @@ def generate_graph(
             return all_logs
         try:
             graph_filter_validation_logs = self._validate_graph_filter(graph_details)
-            prepared_data, log_pool = self._prepare_plot_data(filtered_pool, graph_details)
+            prepared_data: Dict[str, List[Any]] = {key: filtered_pool[key]["values"] for key in filtered_pool.keys()}
+            log_pool = self._log_non_int_float_data(filtered_pool, graph_details)
```

Comment:
```suggestion
            log_pool = self._log_non_int_float_data(filtered_pool, graph_details)
```
I think a more specific name for "log_pool" would be better, something more like "non_numeric_data_logs"